Readme.md

```diff
@@ -179,7 +182,7 @@ Updating the snp-matrix is triggered manually and should be run either weekly or
 2. Mount FSx drive, `fsx-042`;
 2. Run the following command; 
 ```
-./btb-phylo.sh path/to/fsx-017 path/to/fsx-042/share/phyloConsensus --meta_path path/to/fsx-017/ViewBovine/app/raw --with-docker
+./btb-phylo.sh path/to/fsx-017 path/to/fsx-042/share/ViewBovine_consensus --meta_path path/to/fsx-017/ViewBovine/app/raw --with-docker
```

Comment:
could make it clearer that the path/to needs replacing, like {fsx_path} or something